job-store-service/war/src/main/java/dk/dbc/dataio/jobstore/service/param/PartitioningParam.java

```diff
@@ -167,7 +167,8 @@ private boolean isSubmitterDisabled() {
 
     private boolean canBePreviewOnly() {
         return jobEntity.getSpecification().getType() == JobSpecification.Type.TRANSIENT
-                || jobEntity.getSpecification().getType() == JobSpecification.Type.PERSISTENT;
+                || jobEntity.getSpecification().getType() == JobSpecification.Type.PERSISTENT
+                || jobEntity.getSpecification().getType() == JobSpecification.Type.SUPER_TRANSIENT;
     }
```

Comment:
Ville være pænere med en
 previewSet = EnumSet.of(TRANSIENT, PERSISTENT, SUPER_TRANSIENT); // gerne som global final static
return previewSet.contains(jobEntity.getSpecification().getType())